src/mlpack/tests/local_coordinate_coding_test.cpp

```diff
@@ -43,15 +46,18 @@ void VerifyCorrectness(const vec& beta, const vec& errCorr, double lambda)
   }
 }
 
-
-TEST_CASE("LocalCoordinateCodingTestCodingStep",
-          "[LocalCoordinateCodingTest]")
+TEMPLATE_TEST_CASE("LocalCoordinateCodingTestCodingStep",
+    "[LocalCoordinateCodingTest]", arma::mat, arma::fmat)
 {
+  typedef TestType MatType;
+  typedef arma::Col<typename MatType::elem_type> VecType;
+
   double lambda1 = 0.1;
   uword nAtoms = 10;
 
-  mat X;
-  X.load("mnist_first250_training_4s_and_9s.arm");
+  mat inX; // The .arm file is saved as an arma::mat.
```

Comment:
```suggestion
  arma::mat inX; // The .arm file is saved as an arma::mat.
```

Reply:
or MatType, up to you

Reply:
There's already a `using namespace arma` at the top of this test file, so `mat` is fine, and if we extend to Bandicoot later, we can use `coot::mat`.  But if you would rather I drop the `using namespace arma` now, I can do that too, just let me know.

Reply:
it is fine, we can keep the namespace for now.